docs/extensibility/resources.md

```diff
@@ -1,26 +1,21 @@
-# Resource-based extensions
+# Config Map for resource-based extensions
 
-## Introduction
-
-Resource-based extensions are means for users to introduce custom resource pages to Busola. To achieve that, you need to create a Config Map in the `kube-public` Namespace with a label `busola.io/extension=resource`.
-
-The Config Map needs to have several fields created that define how the resources are handled.
+## Overview
 
+This document describes the required Config Map sections, you need to configure in order to handle your CRD UI page.
 All sections can be provided as either JSON or YAML.
 
-## Config Map sections
-
-### resource section
+## resource section
 
 The `resource` section is required and contains basic information about the resource. For example, `kind` and API details.
 
 - **kind** - _[required]_ Kubernetes kind of the resource.
 - **group** - _[required]_ API group used for all requests.
 - **version** - _[required]_ API version used for all requests.
-- **path** - path fragment for this resource used in the url. Defaults to pluralized lowercase `kind`. Used mostly to provide an alternative url to avoid conflicts with other resources.
 - **scope** - either `namespace` or `cluster`. Defaults to `cluster`.
+- **path** - path fragment for this resource used in the url. Defaults to pluralized lowercase `kind`. Used to provide an alternative url to avoid conflicts with other resources.
```

Comment:
```suggestion
- **path** - path fragment for this resource used in the URL. Defaults to pluralized lowercase **kind**. Used to provide an alternative URL to avoid conflicts with other resources.
```